content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -1,60 +1,199 @@
-============================
-Reports for a Better Support
-============================
+=========
+Reporting
+=========
 
-An efficient customer service solution should have a built-in reporting option. Reports allow you to
-track trends, identify areas for improvement, manage employees’ workloads and, most importantly,
-meet your customer’s expectations.
+Odoo *Helpdesk* includes several reports that provide the opportunity to track trends for customer
+support tickets, identify areas for improvement, manage employee workloads, and confirm when
+customer expectations are met.
 
-Cases
-=====
+Available reports
+=================
 
-Some examples of the reports Odoo Helpdesk can generate include:
+Details about the reports available in Odoo *Helpdesk* can be found below. To view the different
+reports, go to :menuselection:`Helpdesk --> Reporting`.
 
-- The number of tickets *grouped by* team and ticket type.
+Ticket Analysis
+---------------
 
-| In this manner, you are able to evaluate which ticket types have been the most frequent ones,
-  plus the workload of your teams.
-| Apply *Time Ranges* if you would like to make comparisons to a *Previous Period* or a *Previous
-  Year*.
+The *Ticket Analysis* report (:menuselection:`Helpdesk --> Reporting --> Ticket Analysis`) provides
+an overview of every customer support ticket in the database. This includes the number of tickets
+assigned among teams and individual users.
 
-.. image:: reports/report_team_ticket_type.png
+This report is useful in identifying where teams are spending the most time, and helps determine if
+there is an uneven workload distribution among the support staff. The default report counts the
+number of tickets per team and groups them by stage.
+
+.. image:: reports/reports-tickets-default.png
```

Comment:
All images: we don't need to prepend `reports-` to the image filename since it is already implied by residing in the `reports/` folder